scipy/stats/tests/test_qmc.py

```diff
@@ -209,39 +211,56 @@ def test_update_discrepancy(self):
                                              r"broadcastable"):
             update_discrepancy(x_new, space_1[:-1], disc_init)
 
+    def test_perm_discrepancy(self):
+        qmc_gen = qmc.LatinHypercube(5)
```

Comment:
This is causing the other test failure in the custom `check_random_state`. Please check the documentation of that one. It says it will turn `int`s into a `Generator` but instead it raises this error.
```
E   ValueError: NumPy 1.16 doesn't have Generator, use either NumPy >= 1.17 or `seed=np.random.RandomState(seed)`
```

Reply:
Indeed, thanks. It was missing explicit seeding to work. It is by design that having `seed=None` only calls `np.random.Generator`. We dit this intentionally so that `None` would give the recommended way only. The documentation is also not talking about `RandomState` on purpose.